tests/copylat.cpp

```diff
@@ -40,6 +40,8 @@ using namespace gdrcopy::test;
 // manually tuned...
 int num_write_iters = 10000;
 int num_read_iters = 100;
+int small_size_iter_factor = 1000;
```

Comment:
I understand the intention and usefulness for small sizes. However, it changes what the number of iterations users specify. Is there a better way to do this or could you provide an explanation message? Currently, the users need to read the code in order to know that small sizes and large sizes use different number of iterations.